addons/binding/org.openhab.binding.mihome/ESH-INF/thing/gateway.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="mihome" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="gateway">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge" />
+		</supported-bridge-type-refs>
+
+		<label>Xiaomi Mi Smart Home Gateway</label>
+		<description>Multifunctional Gateway. One button click turns on/off the device. Two clicks toggle "home security" feature. 
+		Three button clicks allow you to link the Xiaomi Zigbee devices with the Gateway. After you press the button 3 times, the light will flash blue;
+		After that you'll be able to add new sensors by pressing their hole button for 3 seconds.
+		</description>
+
+		<channels>
+			<channel id="brightness" typeId="brightness" />
+			<channel id="illumination" typeId="illumination" />
+			<channel id="color" typeId="color" />
+			<channel id="colorTemperature" typeId="colorTemperature" />
+			<channel typeId="enableSound" id="enableSound"></channel>
```

Comment:
Better close the channel element like in the lines above instead of having a separate `</channel>`.